src/unpage/agent/templates/blank.yaml

```diff
@@ -17,3 +17,28 @@ prompt: >
 #
 tools:
   - "*"
+
+# Customize and override configuration for any plugins the agent uses.
+# config:
```

Comment:
Random thought as I was reading this: I wonder if we should set `extra='forbid'` in the `PartialConfigForAgent` so that the user gets an error if they try to override anything except plugins?